src/runtime/components/forms/Input.vue

```diff
@@ -1,5 +1,5 @@
 <template>
-  <div :class="ui.wrapper">
+  <div :class="(type === 'hidden') ? 'hidden' : ui.wrapper">
```

Comment:
Best of both worlds would be to use array notation:
```suggestion
  <div :class="[ui.wrapper, { hidden: type === 'hidden' }]">
```

Reply:
Thanks @noook.
Just a question for me to understand, before I commit your suggestion: Why would you want the styles of ui.wrapper, if the element is hidden?

Reply:
I haven't paid attention to this, I was just suggesting this syntax so that pre-configured classes and conditional ones could co-exist, but here it is indeed not relevant